daemon/container.go

```diff
@@ -275,6 +275,22 @@ func validateHostConfig(hostConfig *containertypes.HostConfig, platform string)
 	if hostConfig == nil {
 		return nil
 	}
+
+	if hostConfig.Privileged {
+		for _, deviceMapping := range hostConfig.Devices {
```

Comment:
Can you add some comments as to why we are checking these paths?

Reply:
@cpuguy83  I am thinking we don't need to do that check. because its same like mounting a hostdirectory inside the container and writing to it. IMO, it should be only upto the application on were to write files. Docker should not decide that and error out the process.

Also while creating the device mapping, we will exclude that.

Reply:
Removed the host directory check.